opencog/atomspace/TruthValue.h

```diff
@@ -71,6 +71,17 @@ enum TruthValueType
     NUMBER_OF_TRUTH_VALUE_TYPES
 };
 
+//Flags for alternative ways of merging two different truth value types
+//as described in https://github.com/opencog/opencog/issues/1295
+enum TVMergeStyle
+{
+    USE_OLDER_TV_TYPE,
+    USE_NEWER_TV_TYPE,
```

Comment:
> USE_STRONGER_TV_TYPE, is STRONGER_TV // of the two, pick the stronger TV
> which is what the current code does, and it is what Ben doesn't like
>


Actually my problem wasn't about picking the stronger TYPE, but rather
about the way the simpleTV merge function chose the TV with the highest
confidence rather than doing some sort of confidence-weighted
combination...